rst2pdf/tests/input/test_issue_267.txt

```diff
@@ -1,90 +0,0 @@
-Alignment for figures and images
-================================
-
-Unaligned PNG Image:
-
-.. image:: images/biohazard.png 
-
-Align left:
-
-.. image:: images/biohazard.png 
-   :align: left
-
-Align right:
-
-.. image:: images/biohazard.png 
-   :align: right
-
-Align center:
-
-.. image:: images/biohazard.png 
-   :align: center
-
-Unaligned PNG figure:
-
-.. figure:: images/biohazard.png 
-   :figwidth: 50%
-
-Align left:
-
-.. figure:: images/biohazard.png 
-   :align: left
-   :figwidth: 50%
-
-Align right:
-
-.. figure:: images/biohazard.png 
-   :align: right
-   :figwidth: 50%
-
-Align center:
-
-.. figure:: images/biohazard.png 
-   :align: center
-   :figwidth: 50%
-
-Alignment for PDF-based images:
```

Comment:
Could we keep the test with the png images and just remove the PDFs and update the test?

Reply:
Yes. I've restored as `test_alignment_of_figures_and_images`.